dll/win32/shell32/CDefView.cpp

```diff
@@ -884,6 +887,9 @@ PCUITEMID_CHILD CDefView::_PidlByItem(LVITEM& lvItem)
 */
 int CDefView::LV_FindItemByPidl(PCUITEMID_CHILD pidl)
 {
+    if (!m_ListView)
```

Comment:
ok I actually wonder how it could be possible to have those `LV_`-prefixed functions be called when `m_ListView` is NULL. Isn't the code constructed such that this is not possible?

Because if so, having `ASSERT(m_ListView);` might be better, for debugging purposes, in order to catch when those LV functions are called while `m_ListView` is NULL and fix the caller functions instead.

Reply:
I added `ASSERT(m_ListView);` into the `LV_` functions and removed actual check in the `LV_` functions.
The actual checks are done by the outside of the `LV_` functions.